samples/bluetooth/fast_pair/locator_tag/sysbuild/ipc_radio/prj.conf

```diff
@@ -4,16 +4,15 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-CONFIG_IPC_SERVICE=y
-CONFIG_MBOX=y
+CONFIG_SERIAL=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
 
-CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_LOG=y
+CONFIG_LOG_MODE_MINIMAL=y
 
-CONFIG_MAIN_STACK_SIZE=512
-CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
-
-CONFIG_BT=y
-CONFIG_BT_HCI_RAW=y
```

Comment:
Why is that no longer necessary?

Reply:
As described in the commit msg:
```
Removed Kconfig options that are already enabled via added overlay (nrf/applications/ipc_radio/overlay-bt_hci_ipc.conf) when the SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC sysbuild Kconfig option is enabled.
```
Logic that handles adding this overlay is located in the nrf/sysbuild/netcore.cmake

Reply:
oh, my bad. I did not read the commit description. Thanks for explanation

Reply:
Overlay was always added when mentioned sysbuild option was enabled so there was no need to duplicate them, especially that they were applied at the end, after this conf file.

However, after short talk with ipc_radio maintainer, i changed the approach and resigned from using it as it wasn't very flexible due to how the overlays in sysbuild are applied.